server/src/http/controllers/jobs/jobs.controller.ts

```diff
@@ -455,12 +456,19 @@ export class JobsController extends Controller {
     @Query() opcoUrl?: string,
     @Query() @Hidden() useMock?: string
   ): Promise<IApiError | { lbbCompanies: ILbaItem[] } | { lbaCompanies: ILbaItem[] }> {
-    const result = await getJobsQuery({ romes, caller, referer, latitude, longitude, radius, insee, sources, diploma, opco, opcoUrl, useMock })
+    const result = await getJobsQuery({ romes: romes.join(","), caller, referer, latitude, longitude, radius, insee, sources, diploma, opco, opcoUrl, useMock })
 
-    if (result.error) {
+    if ("error" in result) {
       this.setStatus(500)
+      return result
+    }
+    const { matchas } = result
+    if ("results" in matchas) {
+      matchas.results.map((matchaOffre) => {
+        // non bloquant
+        addMatchaSearchView(matchaOffre.job.id)
+      })
```

Comment:
```suggestion
      matchas.results.map((matchaOffre) => addMatchaSearchView(matchaOffre.job.id))
```

Je ne suis pas certain que de mettre "Matcha" soit une bonne chose.Désormais le produit est unifié.
Je mettrais plutot "recruteur"

Reply:
fixed